docs/api/api_blueprint.apib

```diff
@@ -20834,6 +20834,88 @@ Get the task status and the final result of the task as response
         }
 
 
+## Event Types Statistics Details [/v1/admin/statistics/event-types]
+
+**Event Types:**
+
+| Parameter | Description | Type |
+|:----------|-------------|------|
+| `Camp, Treat & Retreat`  | No. of events of type - Camp, Treat & Retreat | Integer |
+| `Dinner or Gala`  | No. of events of type - Dinner or Gala | Integer |
+| `Other`  | No. of events of type - Other | Integer |
+| `Concert or Performance`  | No. of events of type - Concert or Performance | Integer |
+| `Conference`  | No. of events of type - Conference | Integer |
+| `Seminar or Talk`  | No. of events of type - Seminar or Talk | Integer |
+| `Convention`  | No. of events of type - Convention | Integer |
+| `Festival or Fair`  | No. of events of type - Festival or Fair | Integer |
+| `Tour`  | No. of events of type - Tour | Integer |
+| `Screening`  | No. of events of type - Screening | Integer |
+| `Game or Competition`  | No. of events of type - Game or Competition | Integer |
+| `Party or Social Gathering`  | No. of events of type - Party or Social Gathering | Integer |
+| `Race or Endurance Event`  | No. of events of type - Race or Endurance Event | Integer |
+| `Meeting or Networking Event`  | No. of events of type - Meeting or Networking Event | Integer |
+| `Attraction`  | No. of events of type - Attraction | Integer |
+| `Class, Training, or Workshop`  | No. of events of type - Class, Training, or Workshop | Integer |
+| `Appearance or Signing`  | No. of events of type - Appearance or Signing | Integer |
+| `Tournament`  | No. of events of type - Tournament | Integer |
+| `Rally`  | No. of events of type - Rally | Integer |
```

Comment:
This schema doesn't match with what is there right now. Also, you don't have to mention the actual topics here in the doc. You can just put an example response maybe. Since these topics/sub-topics etc are dynamic.

Reply:
Done